source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -349,6 +352,10 @@ Http::FilterDataStatus JsonTranscoderFilter::encodeData(Buffer::Instance& data,
   readToBuffer(*transcoder_->ResponseOutput(), data);
 
   if (!method_->server_streaming() && !end_stream) {
```

Comment:
support or add a TODO for streaming case?

Reply:
:+1: